.github/workflows/check-browser-service-release.yml

```
@@ -107,6 +113,7 @@ jobs:
         if: steps.decide.outputs.act == 'true'
         env:
           BRANCH: ${{ steps.decide.outputs.branch }}
+          CODE_SUMMARY: ${{ steps.code.outputs.summary }}
```

actionlint (1.7.12)

[error] property "code" is not defined in object type {decide: {conclusion: string; outcome: string; outputs: {string => string}}; pin: {conclusion: string; outcome: string; outputs: {string => string}}; pypi: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)